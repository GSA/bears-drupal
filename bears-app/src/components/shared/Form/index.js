function Form({ children }) {
  return <form className="usa-form">{children}</form>
}

export default Form
