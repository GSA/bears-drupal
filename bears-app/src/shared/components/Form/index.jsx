import PropTypes from 'prop-types'
/**
 * a functional component that wraps form elements in a form element
 * @component
 * @param {React.ReactNode} children - inherited children
 * @return {html} returns a semantic html form element, with all its children
 */
function Form({ children }) {
  return <form className="usa-form">{children}</form>
}

Form.propTypes = {
  children: PropTypes.node,
}

export default Form
