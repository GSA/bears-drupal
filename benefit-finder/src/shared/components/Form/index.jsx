import PropTypes from 'prop-types'
/**
 * a functional component that wraps form elements in a form element
 * @component
 * @param {React.ReactNode} children - inherited children
 * @return {html} returns a semantic html form element, with all its children
 */
function Form({ children }) {
  return (
    <form className="benefit-form">
      <div className="grid-container">{children}</div>
    </form>
  )
}

Form.propTypes = {
  children: PropTypes.node,
}

export default Form
