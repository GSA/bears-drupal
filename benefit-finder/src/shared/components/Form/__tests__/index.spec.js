import TestRenderer from 'react-test-renderer'
import Form from '../index.jsx'

describe('Form', () => {
  it('renders a match to the previous snapshot', () => {
    const component = TestRenderer.create(<Form />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
