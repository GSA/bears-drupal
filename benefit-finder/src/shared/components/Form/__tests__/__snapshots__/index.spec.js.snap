// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form renders a match to the previous snapshot 1`] = `
<form
  className="benefit-form"
>
  <div
    className="grid-container"
  />
</form>
`;
